resources/asciidoctor/lib/chunker/breadcrumbs.rb

```diff
@@ -12,9 +12,76 @@ def generate_breadcrumbs(doc, section)
       result = ['<div class="breadcrumbs">']
       result += generate_breadcrumb_links(section).reverse
       result << '</div>'
+      if result[2].to_s.include? 'APM'
+        result[2] = generate_apm_breadcrumbs(doc)
+      end
+      if result[2].to_s.include? 'ECS Logging'
+        result.insert(2, generate_ecslogging_breadcrumbs)
+      end
       Asciidoctor::Block.new doc, :pass, source: result.join("\n")
     end
 
+    def generate_apm_breadcrumbs(doc)
+      title = doc.title
+      short = title.sub(/APM /, '')
+      <<~HTML.strip
+        <span class="breadcrumb-link">
+          <div id="related-products" class="dropdown">
+            <div class="related-products-title">APM:</div>
+            <div class="dropdown-anchor">#{short}<span class="dropdown-icon"></span></div>
+            <div class="dropdown-content">
+              <ul>
+                <li class="dropdown-category">APM</li>
+                <ul>
+                  <li><a id="guide" href="http://localhost:8000/guide/index.html">User Guide</a></li>
+                </ul>
+                <li class="dropdown-category">APM agents</li>
+                <ul>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/android/current/index.html">Android Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/go/current/index.html">Go Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/swift/current/index.html">iOS Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/java/current/index.html">Java Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/dotnet/current/index.html">.NET Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/nodejs/current/index.html">Node.js Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/php/current/index.html">PHP Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/python/current/index.html">Python Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/ruby/current/index.html">Ruby Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/rum-js/current/index.html">Real User Monitoring JavaScript Agent Reference</a></li>
```

Comment:
Do you think relative links would work? The preview sending you to the public docs is confusing.

Reply:
I don't see why not.

Reply:
Testing in https://github.com/elastic/docs/pull/2617